CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## unreleased
 
+## v4.12.0 - 2024.08.16
+
+* Update CSI driver for Kubernetes 1.31
+  Update CSI sidecars to latest
```

Comment:
i think it will be cleaner to have them in bullet points `*` like done below and also link the relevant PRs 🙂

Reply:
They're all in the same PR though

Reply:
Oh it's all going in the same paragraph when rendered. Let me find something better

Reply:
Added newlines so that in the rendered .md those are kept on separate lines
Still a single PR though